### DISCUSSION TITLE
Context menu option to filter by beatmap's title/artist(/tags?)

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#18959] [Ideas]
A little idea I thought might be helpful: add the option to filter by beatmap's title/artist in the carousel context menu, so you don't have to manually type the filter in the search bar, especially with the long ones. Filtering by title might be of little use since the carousel sorts with that by default, but it could be used by users who sort differently.

I mean this context menu:
![image](https://user-images.githubusercontent.com/43073074/176739191-7050bc29-2d9d-4b0a-8da9-6c401f553d8e.png)

Something like "filter with title" and "filter with artist" (or "add title/artist to filter").

I understand there is an option to sort by artist/title, however, I think some users may find the filter more intuitive since it hides everything else

**Comment by peppy** [comment]
probably easier to do this by clicking the title/artist on the left in the details area (you already can for tags, although currently it filters on the beatmap listing overlay rather than song select, which is probably not correct).
